capi/geos_c.h.in

```diff
@@ -1363,6 +1370,12 @@ extern int GEOS_DLL GEOSCoordSeq_getSize(const GEOSCoordSequence* s,
 extern int GEOS_DLL GEOSCoordSeq_getDimensions(const GEOSCoordSequence* s,
     unsigned int *dims);
 
+/*
+ * Check orientation of a Coordinate Sequence.
+ * Return 0 on exception.
+ */
+extern int GEOS_DLL GEOSCoordSeq_isCCW(const GEOSCoordSequence* s, char* val);
```

Comment:
what's the "val" for ? Please document in that comment (meaning of each possible value written in there, and in which case no value is written)

Reply:
Clarified this with comment and better variable name. Thanks.